src/RecoveryServices/RecoveryServices.SiteRecovery.Test/RecoveryServices.SiteRecovery.Test.csproj

```diff
@@ -13,8 +13,9 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.RecoveryServices" Version="4.3.3-preview" />
     <PackageReference Include="Microsoft.Azure.Management.Network" Version="23.0.0" />
-    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="56.0.0" />
-    <PackageReference Include="Microsoft.Azure.Management.RecoveryServices.Backup" Version="5.0.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="57.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.RecoveryServices" Version="4.3.2-preview" />
```

Comment:
```suggestion
    <PackageReference Include="Microsoft.Azure.Management.RecoveryServices" Version="4.3.3-preview" />
```

Reply:
recovery services SDK was (accidently) downgraded

Reply:
thank you